README.md

```diff
@@ -15,7 +15,7 @@ The book kicks off by exploring the basics of Node.js describing its asynchronou
 # Key Features of Node.js  Design Patterns
 
 - Create reusable patterns and modules by leveraging the new features of Node.js 
--	Understand the asynchronous single thread design of node and grasp all its features and patterns to take advantage of various functions
+-	Understand the asynchronous single thread design of node and grasp all its features and patterns to take advantage of various   functions
```

Comment:
Is the only difference the 2 spaces here?
<img width="934" alt="screen shot 2018-03-09 at 10 53 07 am" src="https://user-images.githubusercontent.com/205629/37204268-2703ef96-2388-11e8-8015-fa079e3cb318.png">